WooCommerce/Classes/ViewModels/RefundDetails/RefundDetailsViewModel.swift

```diff
@@ -40,6 +73,41 @@ final class RefundDetailsViewModel {
 }
 
 
+// MARK: - Private methods
+//
+private extension RefundDetailsViewModel {
+    /// Calculate the subtotal for each returned item
+    ///
+    func calculateItemSubtotal() -> NSDecimalNumber {
+        let itemSubtotals = refund.items.map { NSDecimalNumber(string: $0.subtotal) }
```

Comment:
I think that this NSDecimalNumber allocation can be dangerous since I discovered that NSDecimalNumber uses by default the locale decimal separator to evaluate a decimal amount.

For example if an item price is `10,99` with a comma separator, and in your locale decimal numbers are separated by a point, you will get a new decimal number without decimal: `10.00`. 

I discovered this in this PR where I obtained a wrong formatted amount. https://github.com/woocommerce/woocommerce-ios/pull/1698/files#diff-9602fa47bdc00e54d6c27795709e94acR114
I believe this can also affect the calculation of the total amount.

Reply:
Calculating currency is so tricky 🙂
Thank you for bringing this to my attention. Fixing now!

Reply:
✅